common/datablocks/src/data_block.rs

```diff
@@ -132,6 +117,20 @@ impl DataBlock {
         }
     }
 
+    /// Take the first data value of the column.
+    pub fn first(&self, col: &str) -> Result<DataValue> {
+        let series = self.try_array_by_name(col)?;
+        let values = series.to_values()?;
```

Comment:
`to_values()` is too heavy,  try  `try_get(idx)`

Reply:
There is already a method in `DataColumn` named try_get

Reply:
Ok, I will update it in another patch.